build-parent/pom.xml

```diff
@@ -155,6 +157,7 @@
             <plugins>
                 <plugin>
                     <artifactId>maven-surefire-plugin</artifactId>
+                    <version>${surefire-plugin.version}</version>
```

Comment:
This should already be defined by the jboss-parent pom. overriding the version.surefire.plugin should be all that is needed to get the newer version.

Reply:
done